client/trino-cli/src/main/java/io/trino/cli/FormatUtils.java

```diff
@@ -174,7 +174,7 @@ public static String formatFinalTime(Duration duration)
             return formatTime(duration);
         }
 
-        return format("%.2f", (totalMillis / 1000.0));
+        return format("0:%05.2f", (totalMillis / 1000.0));
```

Comment:
or maybe we should change `formatTime(Duration)` and remove the `if (totalMillis >= MINUTES.toMillis(1))` condition

Reply:
Makes sense. We could always render subsecond part.

I've updated this PR so now it renders:

When the query is running:
<img width="849" alt="Screenshot 2022-05-23 at 15 53 42" src="https://user-images.githubusercontent.com/66972/169836039-1a5def80-ab62-4b3c-8078-d8b90f1003d2.png">

When the query completes under a minute:
<img width="529" alt="Screenshot 2022-05-23 at 15 58 29" src="https://user-images.githubusercontent.com/66972/169836201-ffdee2dd-84b3-4163-96cf-0432464ddef2.png">

When the query completes over a minute:
<img width="475" alt="Screenshot 2022-05-23 at 16 35 47" src="https://user-images.githubusercontent.com/66972/169843711-9a89731e-1d32-4b81-8c7c-6e7c8ff01aae.png">

Reply:
> Makes sense. We could always render subsecond part.

I didn't mean that.

In `04:06.89` output, the `.89` is superfluous, isn't it?

Reply:
Does this make user experience worse?

Reply:
For reference, PostgreSQL's `psql` always prints milliseconds with nanoseconds fraction:
```
Time: 1.123 ms
```
and for long queries it also prints the time in minutes:
```
Time: 1234.567 ms (12:34:56)
```

From the UX perspective, in my subjective opinion, the following matters:
* values being easy to interpret - I shouldn't have to _think_ about what's the unit, at the first glance (never assume users _get_ familiar with the CLI, even power users might use it sporadically)
* output being consistent - I should be able to find the same information in the same place when running subsequent queries, no matter how long they take

It's ok to cut off the least significant part for longer running queries. But it's more important for users to know the units. Is it `HH:mm` or `mm:ss`? Adding milliseconds makes it unambiguously `mm:ss.SSS`.

Reply:
Also I think having 3 digits for milliseconds would help.